src/modules/dashboard/components/ActionsPage/StakeRequiredBanner/StakeRequiredBanner.tsx

```diff
@@ -11,7 +11,14 @@ import styles from './StakeRequiredBanner.css';
 const MSG = defineMessages({
   stakeRequired: {
     id: `dashboard.ActionsPage.StakeRequiredBanner.stakeRequired`,
-    defaultMessage: `Motion requires at least 10% stake to appear in the actions list.`,
+    defaultMessage: `{isDecision, select,
+      true {Decision}
+      false {Motion}
+      } requires at least 10% stake to appear in the
+      {isDecision, select,
+        true {Decisions}
```

Comment:
@chinins I think because it is referring to a specific feature it should be capitalised, but i can check with everybody's favourite product guy @arrenv, and change if necessary.